### PR TITLE
daemon-rpc: expand documentation for get_output_distribution

### PR DESCRIPTION
**Pull request by jeffro256**

### CODE REVIEW

**Comment by plowsof** [comment]
its a shame that tables are too advanced for our docs page currently hm

File: resources/developer-guides/daemon-rpc.md [hunk 1]
```diff
@@ -1472,24 +1472,53 @@ $ curl http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"g
 
 ### **get_output_distribution**
 
+Get the number of transaction outputs for given cleartext amounts (0 meaning RingCT) inside each block within a specified range, which is useful for decoy selection. Note that all "height" values below are basically 0-based block indices, like a C-style array, not descriptors of the height of the chain when that block is the top. So the genesis block has "height" 0, and the very highest block has "height" N-1, where N is the total number of blocks in the chain.
 
+Let's say that the blockchain had a state like so:
+
+|---------------|----------------------------------------------------|
```
Comment (plowsof) [inline resources/developer-guides/daemon-rpc.md:1479]:
```suggestion
```

**Comment by jeffro256** [comment]
Will remove table and try to make less verbose.

**Review by plowsof — APPROVED** [review]

**Review by nahuhh — APPROVED** [review]

**Comment by plowsof** [comment]
docs are moving to https://github.com/monero-project/monero-docs

merged in https://github.com/monero-project/monero-docs/pull/68

closing, thank you.

**Pull request closed without merging**